src/LabJackPython.py

```diff
@@ -137,7 +139,6 @@ def _loadLibrary():
             wlib = ctypes.CDLL("labjackud")
         if wlib is not None:
             try:
-                wlib.eGetPtr
```

Comment:
I don't know if this change is harmless or not.  The return value is thrown away but I don't know if the property is read for some other side effects.  This concerns the underlying driver so @davelopez01 should look at this.